docs/2.6.0/docs/daml-finance/concepts/index.rst

```diff
@@ -9,7 +9,7 @@ where each of these concepts is implemented in the library.
 
 The most important definitions are also summarized in the :doc:`glossary <../reference/glossary>`.
 
-.. toctree::
+.. .. toctree::
```

Comment:
Does this mean you're planning to build the TOC via the `_toc.yml` file ?

Reply:
To be clear, [this is already the case](https://github.com/digital-asset/docs.daml.com/blob/f4a3ce5a1033c7be7e485d1ecbaff9d39147be4d/docs/2.6.0/bin/setup-sphinx-source-tree#L60-L64). After extracting the tarball, we run
```
# Disable all ToCs
find $SPHINX_DIR/source -type f -name '*.rst' -print0 | while IFS= read -r -d '' file
do
    sed -i 's|.. toctree|.. .. toctree|g' $file
done
```
on the whole source tree. The build actually fails without that step; if there's a toc entry anywhere the single-file toc plugin we use errors out.